src/main/java/com/tstepnik/planner/domain/Statistics.java

```diff
@@ -0,0 +1,72 @@
+package com.tstepnik.planner.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.net.Inet4Address;
+
+@Entity
+public class Statistics {
```

Comment:
How are you going to update this stats?

Reply:
Every time when clinet request stats, get new object which is created in service where some specific methods retrieve current data from DB.

Reply:
Which means we will create new stats objects (and save it) every time user reloads the page? Don't you think it is a little bit too much?